tests/xlsx/xlsx_export.test.ts

```diff
@@ -936,7 +958,7 @@ describe("Test XLSX export", () => {
       }
     );
 
-    test.each(["line", "scatter", "bar", "combo"])(
+    test.each(["line", "scatter", "bar", "combo", "radar"])(
```

Comment:
We could discuss the meaning of "bloating" here because as you mention it's just snapshots, it's not making the codebase bigger, nor worse to understand. The true ideal - load the xlsx file in excel and check that it's behaving properly,  obviously we can't do this (fast and easy at least). The snapshot of the xlsx file allows to check the end of the pipe. 
If I understand well, your point would be that we could make one gigantic test that have EVERYTHING in it, export and snapshot it?

Reply:
After discussing with Rémi: one snapshot test for each type of chart sounds fine. No need to have 10 snapshots/type of chart.